kyuubi-common/src/main/scala/org/apache/kyuubi/session/SessionManager.scala

```diff
@@ -66,11 +66,11 @@ abstract class SessionManager(name: String) extends CompositeService(name) {
   def closeSession(sessionHandle: SessionHandle): Unit = {
     _latestLogoutTime = System.currentTimeMillis()
     val session = handleToSession.remove(sessionHandle)
-    if (session == null) {
-      throw KyuubiSQLException(s"Invalid $sessionHandle")
+    if (session != null) {
+      session.close()
+    } else {
+      warn(s"$sessionHandle is closed, current opening sessions $getOpenSessionCount")
```

Comment:
this belongs to if-branch

Reply:
Sorry, I don't know how to fix it. Please give me some guidance.